workflow/tests/hpxml_translator_test.rb

```diff
@@ -830,6 +831,7 @@ def _verify_simulation_outputs(runner, rundir, hpxml_path, results)
       end
       query = "SELECT SUM(Value) FROM TabularDataWithStrings WHERE ReportName='EnvelopeSummary' AND ReportForString='Entire Facility' AND TableName='Opaque Exterior' AND (RowName='#{wall_id}' OR RowName LIKE '#{wall_id}:%' OR RowName LIKE '#{wall_id} %') AND ColumnName='Net Area' AND Units='m2'"
       sql_value = UnitConversions.convert(sqlFile.execAndReturnFirstDouble(query).get, 'm^2', 'ft^2')
+      assert_operator(sql_value, :>, 0.01)
```

Comment:
Good idea! Should there be similar checks for other surface types? Windows, doors, slabs, ...